src/Robo/Plugin/Commands/DeployCommands.php

```diff
@@ -73,5 +84,58 @@ public function deployDrupal(string $appDirPath, string $siteName = 'default', s
             // https://github.com/drush-ops/drush/issues/2449#issuecomment-708655673
             ->exec("$appDirPath/vendor/bin/drush config:import --yes")
             ->run();
+        // @todo Do we want/need this option if we can just check for environment variables?
+        // @todo This could be useful if a site wanted to opt out of this functionality.
+        // @todo Perhaps we should rename this option then to be more generic?
+        if ($opts['tugboat']) {
+            $this->notifySlackOnFailedBasePreviewBuild($result);
+        }
+        return $result;
+    }
+
+    /**
+     * Notify Slack if a base preview build failed.
+     *
+     * @param \Robo\Result $result
+     *   The result of the task to check.
+     *
+     * @see https://docs.tugboatqa.com/starter-configs/code-snippets/slack-integration/
+     */
+    protected function notifySlackOnFailedBasePreviewBuild(Result $result): void
+    {
+        // Confirm we are in a Tugboat environment.
+        if (getenv('TUGBOAT_PREVIEW_ID') === false) {
+            return;
+        }
+        // Determine if we are building a base preview.
+        // @todo Remove the testing override condition at the end before merging.
```

Comment:
🚧 Remove this testing override before merging. ⚠️

Reply:
Seems like this override could be useful. Why not keep it and make it an option?

Reply:
I'll see what I can do. Standby!